plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -511,4 +512,11 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
             }
         };
     }
+
+    private static void throwIfNotNull(String comment)
```

Comment:
method signature (`throwIfNotNull(String)`) looks generic, but method behavior is not -- it's for comments only.

i'd inline.